packages/flutter/lib/src/material/material.dart

```diff
@@ -423,13 +425,13 @@ abstract class InkFeature {
   /// To add an ink feature to a piece of Material, obtain the
   /// [MaterialInkController] via [Material.of] and call
   /// [MaterialInkController.addInkFeature].
-  InkFeature({
-    this.renderer,
+  InkFeature._({
+    _RenderInkFeatures renderer,
     this.referenceBox,
     this.onRemoved
-  });
+  }) : _renderer = renderer;
 
-  final _RenderInkFeatures renderer;
+  final _RenderInkFeatures _renderer;
```

Comment:
The fact that `_RenderInkFeatures` is private implies that you weren't able to create one already.